### ISSUE TITLE
World Editor crashing when opening huge maps

### ISSUE THREAD

**Issue by Geovane2299** [mmatyas/supermariowar#146]
[I've](url) made some testing in a 380 x 600 world, which seemed somewhat stable (even though when a bot tried to use a warp, the game crashed all the times, but I assumed it would work if I avoided it). But after creating a few areas, and saving, I've tested it out later but the game couldn't read the file anymore. Upon opening the World Editor, it happened the same when I tried creating bigger worlds: many graphical errors (mostly red "T"s as when you win a Level in the red team). All levels were deleted from the world, so it was completely lost.
Tl;dr the game crashes and world gets corrupted when you add a considerable amount of tiles on huge maps.
Link to download world file: https://mega.nz/file/HREySSKZ#YR6PW8aINRqXS026vfXRf2KoIOWZHx8VLHbQTMP8l6o

**Comment by mmatyas** [comment]
Ok, world loading should be working now, [here's the latest build](https://github.com/mmatyas/supermariowar/releases/tag/2020-04-25).

As for crashing *during* the gameplay, that sounds like a separate bug -- it might got fixed but if it does happen again, feel free to report it!
